charts/linkerd-control-plane/values-ha.yaml

```diff
@@ -53,5 +53,8 @@ webhookFailurePolicy: Fail
 # service profile validator configuration
 spValidatorResources: *controller_resources
 
+# policy controller configuration
+policyControllerResources: *controller_resources
```

Comment:
These changes are duped from #11306. We'll want to either remove these from the branch or merge that PR first and then merge main into this branch...